packages/chat-stateful-client/package.json

```diff
@@ -12,7 +12,7 @@
     "build:watch": "rushx build",
     "clean": "rimraf dist && rimraf preprocessed",
     "preprocess": "cpx \"./src/**\" ./preprocessed && babel ./src --out-dir ../preprocessed --extensions \".ts,.tsx\" --keep-file-extension --config-file ./.babelrc.js --relative && rimraf ../preprocessed",
-    "build:by-flavor": "rushx clean && npm run build:esm && rushx build:cjs && api-extractor run --local",
+    "build:by-flavor": "rushx clean && npm run build:esm && rushx build:cjs && npm run api-extractor:by-flavor",
```

Comment:
I think this should be `_build:by-flavor` and `_api-extractor:by-flavor`, because we don't expect developers to ever run these commands directly in their workflow.